### ISSUE TITLE
NuGet package

### ISSUE THREAD

**Issue by aloisdeniel** [Ourpalm/ILRuntime#71]
It should be really easier to integrate `ILRuntime` from [NuGet](https://www.nuget.org/).

I would use a [Cake](https://cakebuild.net/) script, with a [.nuspec](https://docs.microsoft.com/en-us/nuget/schema/nuspec) for automating release process.

**Comment by liiir1985** [comment]
You're right, it's lot easier to interate ILRuntime into project with Nuget. But I'll see how to make it work with Unity, since Unity is the main target of ILRuntime.

**Comment by qiang-todo** [comment]
@liiir1985 你好，借这个issue问一下，把ILRuntime打成Dll在Unity的il2cpp模式下会报错，这个问题会尝试去处理下不？

**Comment by liiir1985** [comment]
@qiang-todo 在Unity中请使用源码，因为里面有针对开发期和发布期不同的条件编译，此外打成DLL打包il2cpp报错是unity的bug，没有办法修复，也没有办法绕开，绕开后iOS64位系统会闪退，因为IL2CPP错误将指针地址编译成了32位整型

**Comment by qiang-todo** [comment]
@liiir1985  好的，明白了，非常感谢 ; )

**Comment by BoysheO** [comment]
此问题是否可以通过使用宏开关分别编译来解决？考虑在说明文档加入相关说明，引导用户
